mixer/pkg/cache/lruCache.go

```diff
@@ -167,29 +168,33 @@ func (c *lruCache) evictExpired(t time.Time) {
 	n := t.UTC().UnixNano()
 	atomic.StoreInt64(&c.baseTimeNanos, n)
 
-	for i := 1; i < len(c.entries); i++ {
+	for i := int32(1); i < int32(len(c.entries)); i++ {
```

Comment:
can use maxEntries instead of computing len ?

Reply:
len on a slice is a memory read, there's no "computing" involved.